pkg/kapp/yttresmod/wait_rule_contract_v1.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
i think this file should be in resourcesmisc package since it's all about waiting.